scale-out-relay/src/main/scala/com/normation/plugins/scaleoutrelay/ScaleOutRelayPluginDef.scala

```diff
@@ -48,4 +56,16 @@ class ScalaOutRelayPluginDef(override val status: PluginStatus) extends DefaultP
   def oneTimeInit : Unit = {}
 
   val configFiles = Seq()
+
+  override def apis: Option[LiftApiModuleProvider[_ <: EndpointSchema]] = Some(ScalaOutRelayConf.api)
+
+
+  override def pluginMenuEntry: Option[Menu] = {
```

Comment:
You should remove the entry menu if there is no page linked to it

Reply:
Should I remove it or create the page (maybe not in this PR) ?

Reply:
remove it